### DISCUSSION TITLE
为什么反复下载后还是没有这个文件

### DISCUSSION THREAD

**Discussion by BaiyeH2O** [mikumifa/biliTickerBuy#279] [提问专区]
![捕获2](https://github.com/mikumifa/biliTickerBuy/assets/174228042/c136221a-d170-4fee-9ed0-851041a4e511)

![捕获](https://github.com/mikumifa/biliTickerBuy/assets/174228042/3c2b6cda-ca2b-467e-82d5-324430a3e3d0)

**Comment by mikumifa** [comment]
win7不支持。。。

**Comment by mikumifa** [comment]
最低也要win10

**Reply by BaiyeH2O** [reply]
知道了
